### PR TITLE
USF-1052: Cart feature matrix

### PR DESCRIPTION
**Pull request by jeff-matthews**
This pull request (PR) adds a table showing which Adobe Commerce features that the cart dropin supports.

>[!NOTE]
>The `OptionsTable` component doesn't currently support image links. It would be nice to have icons/.svg files for each status similar to [this](https://developer.adobe.com/commerce/pwa-studio/integrations/adobe-commerce/features/).

I know that we're attempting to finalize the dropin doc template (#41), but since we'll have to refactor the cart docs after that anyway, it seems prudent to get this info out on production. It's unclear how long it will take to get the template finalized.

### CODE REVIEW

**Review by bdenham — APPROVED** [review]
Nice! Looks great. Thanks!

**Review by keharper — APPROVED** [review]

File: src/content/docs/dropins/cart/index.mdx [hunk 1]
```diff
@@ -7,12 +7,37 @@ sidebar:
...
 
 import Aside from '@components/Aside.astro';
+import OptionsTable from '@components/OptionsTable.astro';
 
 The Cart dropin provides a variety of fully-editabled controls to help you view, update, and merge the products in your cart and mini-cart, including image thumbnails, pricing, descriptions, quantities, estimated shipping and taxes, order summary, merging guest and authenticated cart, and more.
 
-<Aside type="wip" title="In Development">
-  The Cart dropin is in active development. Stay tuned for updates.
-</Aside>
+The following table provides an overview of the Adobe Commerce features that the Cart dropin supports:
```
Comment (bdenham) [inline src/content/docs/dropins/cart/index.mdx:16]:
```suggestion
## Commerce features supported

The following table provides an overview of the Adobe Commerce features that the Cart dropin supports:
```

**Review by bdenham — APPROVED** [review]
Suggested same heading for this table as in Checkout feature matrix.

**Comment by jeff-matthews** [comment]
I converted to the default Starlight table component and added badges. It required some minor style adjustments, which I [made](https://github.com/commerce-docs/microsite-commerce-storefront/commit/f121f7f02a121378a2ffe8f7397e974f505207ab) in a separate integration branch (now the base branch for this PR):

cart

**Review by bdenham — APPROVED** [review]
Excellent use of the Badge component.

**Pull request merged**